pom.xml

```diff
@@ -332,25 +331,25 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-assembly-plugin</artifactId>
-        <version>2.2</version>
+        <version>3.1.0</version>
       </plugin>
 
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
-        <version>2.3.1</version>
+        <version>3.1.1</version>
```

Comment:
Latest is `3.1.2` released a few weeks ago.

https://search.maven.org/search?q=g:org.apache.maven.plugins%20AND%20a:maven-jar-plugin&core=gav

Reply:
Should be `3.1.2` then.